src/playlist/CMakeLists.txt

```diff
@@ -117,39 +118,39 @@ install(DIRECTORY api/libprojectM
 include(CMakePackageConfigHelpers)
 
 write_basic_package_version_file(
-        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/libprojectMPlaylistConfigVersion.cmake"
+        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/projectM4PlaylistConfigVersion.cmake"
         VERSION ${PROJECT_VERSION}
         COMPATIBILITY AnyNewerVersion
 )
 
-configure_package_config_file(libprojectMPlaylistConfig.cmake.in
-        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/libprojectMPlaylistConfig.cmake"
-        INSTALL_DESTINATION "${PROJECTM_LIB_DIR}/cmake/libprojectMPlaylist"
+configure_package_config_file(projectM4PlaylistConfig.cmake.in
+        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/projectM4PlaylistConfig.cmake"
+        INSTALL_DESTINATION "${PROJECTM_LIB_DIR}/cmake/projectM4Playlist"
         PATH_VARS PROJECTM_BIN_DIR PROJECTM_INCLUDE_DIR
         )
 
 install(FILES
-        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/libprojectMPlaylistConfigVersion.cmake"
-        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/libprojectMPlaylistConfig.cmake"
-        DESTINATION "${PROJECTM_LIB_DIR}/cmake/libprojectMPlaylist"
+        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/projectM4PlaylistConfigVersion.cmake"
+        "${CMAKE_CURRENT_BINARY_DIR}/libprojectMPlaylist/projectM4PlaylistConfig.cmake"
+        DESTINATION "${PROJECTM_LIB_DIR}/cmake/projectM4Playlist"
         COMPONENT Devel
         )
 
 install(EXPORT libprojectMPlaylist
-        FILE libprojectMPlaylistTargets.cmake
+        FILE projectM4PlaylistTargets.cmake
         NAMESPACE libprojectM::
-        DESTINATION "${PROJECTM_LIB_DIR}/cmake/libprojectMPlaylist"
+        DESTINATION "${PROJECTM_LIB_DIR}/cmake/projectM4Playlist"
         COMPONENT Devel
         )
 
 # pkg-config export, only supported on UNIX systems.
 if(UNIX)
     include(GeneratePkgConfigFiles)
 
-    set(PKGCONFIG_PACKAGE_NAME "libprojectM-playlist")
+    set(PKGCONFIG_PACKAGE_NAME "projectM-4-playlist")
     set(PKGCONFIG_PACKAGE_DESCRIPTION "projectM Playlist Library")
-    set(PKGCONFIG_PACKAGE_REQUIREMENTS_RELEASE "libprojectM")
+    set(PKGCONFIG_PACKAGE_REQUIREMENTS_RELEASE "projectM-4")
     set(PKGCONFIG_PACKAGE_REQUIREMENTS_DEBUG "libprojectM-debug")
```

Comment:
This line also needs to be changed to
`    set(PKGCONFIG_PACKAGE_REQUIREMENTS_DEBUG "projectM-4-debug")`
Otherwise it will not find the package dependency.

Reply:
@kblaschke great catch, the `Requires:` entry was indeed broken in file `projectM-4-playlist-debug.pc`.
I have pushed a dedicated commit to fix that very issue just now. To reproduce:

### Before
```console
# cd "$(mktemp -d)"
# git clone --depth 1 --branch fix-collisions-with-projectm-prior-to-4-0 https://github.com/hartwork/projectm
# cd projectm
# cmake -S . -B build-ninja-multi -G 'Ninja Multi-Config'
# cmake --build build-ninja-multi --config Debug --parallel
# cmake --build build-ninja-multi --config Release --parallel
# DESTDIR="$PWD"/ROOT_DEBUG/ cmake --install build-ninja-multi --config Debug
# DESTDIR="$PWD"/ROOT_RELEASE/ cmake --install build-ninja-multi --config Release
# find ROOT_*/ -name \*.pc -exec grep -E '^(Libs|Requires):' {} +
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-playlist-debug.pc:Requires:  libprojectM-debug  # <-- BROKEN!
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-playlist-debug.pc:Libs: -L${libdir} -l:projectM-4-playlistd 
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-debug.pc:Requires: opengl 
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-debug.pc:Libs: -L${libdir} -l:projectM-4d  -l/usr/lib/gcc/x86_64-pc-linux-gnu/11/libgomp.so -l/usr/lib64/libpthread.a
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4-playlist.pc:Requires:  projectM-4
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4-playlist.pc:Libs: -L${libdir} -l:projectM-4-playlist 
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4.pc:Requires: opengl 
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4.pc:Libs: -L${libdir} -l:projectM-4  -l/usr/lib/gcc/x86_64-pc-linux-gnu/11/libgomp.so -l/usr/lib64/libpthread.a
```

### After
```console
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-playlist-debug.pc:Requires:  projectM-4-debug
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-playlist-debug.pc:Libs: -L${libdir} -l:projectM-4-playlistd 
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-debug.pc:Requires: opengl 
ROOT_DEBUG/usr/local/lib64/pkgconfig/projectM-4-debug.pc:Libs: -L${libdir} -l:projectM-4d  -l/usr/lib/gcc/x86_64-pc-linux-gnu/11/libgomp.so -l/usr/lib64/libpthread.a
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4-playlist.pc:Requires:  projectM-4
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4-playlist.pc:Libs: -L${libdir} -l:projectM-4-playlist 
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4.pc:Requires: opengl 
ROOT_RELEASE/usr/local/lib64/pkgconfig/projectM-4.pc:Libs: -L${libdir} -l:projectM-4  -l/usr/lib/gcc/x86_64-pc-linux-gnu/11/libgomp.so -l/usr/lib64/libpthread.a
```